src/Components/CookieNotice.test.js

```diff
@@ -141,12 +146,16 @@ describe('CookieNotice', () => {
 
   describe('should set cookie', () => {
```

Comment:
Can you replace this test suite into a single test `it('should set cookie', () => {})` that mock : 
```js
mock(validateCookieExpiration).mockImplementation(() => 'mockedCookieExpiration')
mock(validateCookieName).mockImplementation(() => 'mockedCookieName')
```